Martium.TravelInfo.App/Forms/TravelInfoForm.cs

```diff
@@ -96,6 +97,17 @@ private void MapContributorLinkLabel_LinkClicked(object sender, LinkLabelLinkCli
             System.Diagnostics.Process.Start("http://www.openstreetmap.org");
         }
 
+        private void CountryIsoComboBox_SelectedIndexChanged(object sender, EventArgs e)
```

Comment:
remove this code as it won't be needed when we will show value in combo box instead of key